index.css

```diff
@@ -0,0 +1,167 @@
+*,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+li {
+  margin: 0px;
+  padding: 0px;
+}
+body {
+  font-family: Arial, sans-serif;
+  background: #f5f5f5;
+}
+
+#todo-app {
+  max-width: 913px;
+  max-height: 517px;
+  margin: auto;
+
+  background: white;
+  padding: 20px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  
+}
+#todo-app h1 {
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 400;
+  font-style: normal;
+  font-variation-settings: "slnt" 0;
+  color: black;
+
+  font-size: 48px;
+
+  line-height: 58.09px;
+  text-align: left;
+}
+
+#todo-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.todo-item {
+  background-color: #ffea9e;
+  border: 1px solid #ddd;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  max-width: 380px;
+  min-height: 37px;
+  align-items: center;
+  padding-left: 10px;
+}
+.todo-item:nth-child(odd) {
+  background-color: #fca3a3;
+}
+.todo-item div:nth-of-type(2) {
+  display: flex;
+  flex-direction: column;
+}
+.todo-item div:nth-of-type(2) label {
+  font-family: Inter;
+  font-size: 10px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 12.1px;
+  text-align: left;
+}
+.todo-item div:nth-of-type(2) label:nth-of-type(1)
+{
+    font-style: normal;
+    font-weight: bold;
+   
+    
+} 
+.todo-item div:nth-of-type(1) label {
+  font-family: Inter;
+  font-size: 12px;
+  font-weight: 700;
+  line-height: 14.52px;
+  text-align: left;
+}
+
+.todo-item label {
+  flex-grow: 1;
+}
+
+.todo-item.completed label {
+  text-decoration: line-through;
+  color: #888;
+}
+
+.todo-item input[type="checkbox"] {
+  margin-right: 10px;
+}
+
+#new-todo-title,
+#new-todo-description {
+  width: 205px;
+  height: 22px;
+  top: -81px;
+  left: -264px;
+  gap: 0px;
+  border-radius: 2px 0px 0px 0px;
+  border: 1px 0px 0px 0px;
+  opacity: 0px;
+}
+
+#add-todo {
+  width: 80px;
+  height: 22px;
+
+  padding: 3px 8px 3px 8px;
+  gap: 10px;
+  border-radius: 2.5px 0px 0px 0px;
+
+
+  font-family: Arimo;
+  font-size: 13.33px;
+  font-weight: 400;
+  line-height: 15.33px;
+  text-align: left;
+  margin-top: 10px;
+}
+
+label {
+  text-align: start;
+
+  font-family: Inter;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19.52px;
+  text-align: left;
+}
+
+#todo-list .todo-item {
```

Comment:
Generally it is not great to rely on id for styling, we prefer to use css classes so we can use multipe classes on one element. 

-1